# Transmissão por Correias
Este material serve apenas como um guia para estudos e para as aulas, não substituindo a leitura dos livros indicados e é de uso exclusivo dos alunos do CEFET/RJ campus Angra dos Reis.

A necessidade de transmissão de potência entre eixos distintos é uma constante nos sistemas e dispositivos mecânicos em geral.

Vantagens


-  menor custo inicial
- flexibilidade para grandes distâncias entre centros
- funcionamento silencioso
- capacidade de absorção de choques e sobrecargas
- facilidade para montagem e manutenção
- rendimentos na faixa de 95% a 98%

Desvantagens:


- alongamento permanente da correia
- variação do coeficiente de atrito com a poeira, detritos ou umidade
- necessidade de controle periódico da tensão de montagem
- escorregamento existente na transmissão, caracterizando relações de velocidade não constantes (exceto para correias dentadas)


## Tipos de Correias


- correias planas
- correias trapezoidais ou em V
- correias sincronizadoras ou dentadas
- correias redondas
- correias hexagonais

![](figuras/fig4-1.png)

Quase todos os tipos de correias apresentam basicamente a mesma configuração:


- cabos ou elementos de tração, confeccionados em tiras de poliamida ou poliéster, cordões de fibra de vidro ou cordões de aço, os quis são responsáveis diretamente pela sustentação dos esforços de tração, pelo qual as correias são submetidas quando em funcionamento
- elementos intermediários de preenchimento, confeccionados normalmente em algodão ou náilon, impregnado com borracha, visando dar flexibilidade ao sistema
- capa ou envelope externo, confeccionados tipicamente em borracha sintética, normalmente o neopreme, objetivando aumentar a resistência ao óleo, ao calor e às impurezas típicas da própria transmissão.

![](figuras/fig4-2.png)

## Parâmetros Gerais das Transmissões por Correias

![](figuras/fig4-3.png)

![](figuras/fig4-4.png)

![](figuras/fig4-5.png)

Distância entre eixos:

![](figuras/fig4-6.png)

![](figuras/fig4-7.png)

- Pode apresentar qualquer valor numérico correspondente, devendo, porém, ser compatível com as dimensões das polias envolvidas
- Na ausência de informações, o projetista pode usar os seguintes valores recomendados

$a = dp_2$     

ou      
$a = \dfrac{dp_2 + 3 dp_{1}}{2}$

Nessas circunstâncias, deve-se optar pelo maior valor numérico entre os dois calculados.


Ângulo de abraçamento ou de contato


- correias abertas

$\theta = \pi \pm 2 arcsen \left( \dfrac{dp_2 - dp_1}{2a} \right)$
$+$ → para o ângulo de abraçamento da polia maior
$-$ → ângulo de abraçamento da polia menor


- Para o caso de correias cruzadas

$\theta = \pi + 2 arcsen \left( \dfrac{dp_2 + dp_1}{2a} \right)$

Comprimento da correia


- correias abertas

$L = 2a + \dfrac{\pi}{2} (dp_2 + dp_1) + \dfrac{(dp_2 - dp_1)^2}{4a}$


- Correias cruzadas

$L = 2a + \pi/2 (dp_2 + dp_1) + \dfrac{(dp_2 + dp_1)^2}{4a}$

Relação de Transmissão

$i = \dfrac{dp_1}{dp_2} = \dfrac{rp_1}{rp_2} = \dfrac{Mt_1}{Mt_2} = \dfrac{n_2}{n_1}$

![](figuras/fig4-8.png)

[https://youtu.be/PEq5_b4LWNY](https://youtu.be/PEq5_b4LWNY)

## Forças atuantes sobre a Correia

![](figuras/fig4-9.png)

![](figuras/fig4-10.png)

$\sum F_t = 0$ 

$\mu dN + F cos \left(\dfrac{d \theta}{2} \right) - (F + dF) cos \left( \dfrac{d \theta}{2} \right) = 0$

$\mu dN - dF cos \left(\dfrac{d \theta}{2} \right) = 0$

$\sum F_r = 0$

$dS + dN - Fsen \left( \dfrac{ d \theta}{2} \right) - (F + dF)sen \left(\dfrac{d \theta}{2} \right) = 0$

$dS + dN - 2F sen \left( \dfrac{d \theta}{2} \right) - dF sen \left( \dfrac{ d \theta}{2} \right) = 0$

Considerando $d \theta << 1$ → $sen d \theta = d  \theta$ e $cos d \theta = 1$


$\mu dN - dF = 0$

$dS + dN - F d \theta = 0$

Substituindo $dN$ na primeira equação:

$\mu (F d \theta - d S) - d F = 0$

= $\dfrac{dF}{d \theta} - \mu F = - \mu dS$

Integrando essa equação e aplicando as condições de contorno:

$\theta = 0$ → $F = F_2$
$\theta = \theta_1$ → $F = F_1$

$\dfrac{F_1 - S}{F_2 - S} = e^{\mu \theta_1}$

Força centrífuga

Para o caso de uma correia plana com seção transversal ($b \times t)$:

$S = \gamma b t \dfrac{v^2}{g}$

A força centrífuga, em aplicações gerais, nas quais a velocidade tangencial da correia seja inferior a 15 m/s, pode ser desprezada por sua pequena contribuição em relação aos resultados obtidos para $F_1$ e $F_2$.


- Velocidades tangenciais elevadas para a correia sugerem ainda a utilização de polias com diâmetros maiores, visando a não flexionar a correia de forma excessiva em torno das polias

$v = \omega r p = \dfrac{\pi d_p n}{60}$

Em termos práticos, $v$ não pode ultrapassar a faixa dos 20m/s

## Força líquida e pré-carga sobre a correia

→ A transmissão de potência por uma correia somente torna-se possível caso exista uma tração inicial sobre a mesma.


    → Essa tração inicial, chamada de pré-carga, não deve apresentar valor muito elevado para não avariar a correia por essa tensão excessiva, como também não deve solicitar de forma exagerada os eixos e os mancais correspondentes.


    → Ela também não pode ser muito baixa para não ocasionar o deslizamento excessivo e a “patinação” consequente da correia sobre a polia


![](figuras/fig4-11.png)

$F_1 + F_2 = 2F_i + 2S$ → $F_i = \dfrac{F_1 + F_2}{2} - S$

$F_1 - F_2 = \dfrac{2M_t}{dp} = \dfrac{M_t}{r_p}$ → $F_1 - F_2 = F_t$

onde $F_t$ é a força líquida tangencial

Dispositivos específicos adotados para manter o valor desse carregamento inicial são ilustrados a seguir:

![](figuras/fig4-12.png)

![](figuras/fig4-13.png)

![](figuras/fig4-14.png)

![](figuras/fig4-15.png)

## Variação dos esforços atuantes sobre a correia durante um ciclo completo

![](figuras/fig4-16.png)

![](figuras/fig4-17.png)

## Forças Atuantes sobre o Eixo de Transmissão

![](figuras/fig4-18.png)

$F_H = (F_1 + F_2)cos \beta$

$F_V = (F_1 - F_2) sen \beta$

$\theta_1 + 2 \beta = 180^\circ$

## Tensões Atuantes sobre uma Correia

![](figuras/fig4-19.png)

A tensão devida à força centrífuga é mantida constante ao longo de todo o percurso da correia. As tensões de flexão atuam ao redor das polias, sendo maior (em módulo) para o caso da polia menor. A tensão oriunda da força de tração na correia apresenta variação ao passar ao redor de ambas as polias, devido à passagem de $F_1$ para $F_2$.

## Correias Planas

![](figuras/fig4-20.png)

![](figuras/fig4-21.png)

![](figuras/fig4-22.png)

**Fator de Serviço**

![](figuras/fig4-23.png)

- Para o dimensionamento de uma correia plana são necessárias informações acerca das características do material da correia, as quais são fornecidas normalmente na forma de uma carga por largura de correia ($F_P$)

$(F_1)_{adm} = \dfrac{bF_P}{K_o}$

Como tais correias são disponibilizadas com espessuras padrões, normalmente caberá ao projetista a definição de uma largura mínima de correia correspondente a determinada condição de projeto.

**Exemplo 1:**
Para a transmissão ilustrada utilizando uma correia plana, determinar os esforços atuantes nos ramos da correia: 

![](figuras/fig4-24.png)


## Correias Trapezoidais


- As correias trapezoidais ou em V são fabricadas em comprimentos padronizados e com as dimensões da seção transversal também padronizadas.
- Diferentemente do projeto de uma correia plana, na qual se trabalha com uma única correia, o projeto de correias trapezoidais envolve a definição do tipo de perfil mais adequado como também do número de correias a ser aplicado em determinada situação do projeto.

![](figuras/fig4-25.png)

 Os esforços atuantes em uma correia trapezoidal são semelhantes aos que atuam nas correias planas

$(dN)_{plana} \rightarrow \dfrac{(dN)_{V}}{sen \psi} = \dfrac{(dN)_V}{sen 18^\circ} \rightarrow (dN)_V = 3.2(dN)_{plana}$


- Em termos de projeto, utiliza-se um coeficiente de atrito efetivo, o qual é fornecido pelo fabricante

$\mu' = \mu / sen \psi$

$\mu' = 0.5123$

$\dfrac{F_1 - S}{F_2 - S} = e^{0.5123 \theta_1}$

$S = 2,3562 \left( \dfrac{V}{2.4} \right)^2$        [N]


Seções Trapezoidais padronizadas

→ A, B, C,D e E ← aplicações gerais  
→ 3V, 5V e 8V ← aplicações de carregamentos pesados


**Etapas de seleção e projeto de correias trapezoidais**

Precisamos saber:  
a) potência a transmitir  
b) rotação dos eixos  
c) características do acionamento  
d) estimativa da distância entre eixos  

Etapas:


- Passo 01 - Definição da potência de projeto

$P_p = K_o N_o$

$K_o$ → Para situações nas quais as condições operacionais não sejam conhecidas ou claramente identificadas, deve-se adotar esse valor igual à unidade.


- Passo 02 - Definição do perfil da correia

![](figuras/fig4-26.png)

- Passo 03 - Definição dos diâmetros das polias 

→ não havendo informações de projeto acerca do diâmetro primitivo da polia motora, adotar o diâmetro mínimo recomendado para o perfil da correia selecionada. A partir da relação de transmissão determina-se o diâmetro da polia movida.


- Passo 04 - Definição da velocidade tangencial da correia

$v = \dfrac{\pi d n}{60}$


- Passo 05 - Definição do comprimento da correia e do ângulo de abraçamento da polia

- Passo 06 - Definição do comprimento padronizado da correia e criação da distância entre centros. 

$a_{corr} = a - \left( \dfrac{L - L_{tab}}{2} \right)$

$a_{corr}$ ← distância entre centros corrigida

![](figuras/fig4-27.png)

![](figuras/fig4-28.png)

![](figuras/fig4-29.png)

![](figuras/fig4-30.png)

- Passo 07 - Determinação dos fatores de correção

→ as correias padronizadas são definidas para algumas circunstâncias de ângulos de contato e comprimentos, para os quais se torna necessário efetivar pequenos ajustes a partir de fatores de correção específicos

![](figuras/fig4-31.png)

![](figuras/fig4-32.png)

- Passo 08 - Determinação do HP básico e do HP adicional por correia

$HP_{class} = HP_{bas} + HP_{adic}$


- Passo 09 - Determinação do HP efetivo e do número de correias

→ A multiplicação do HP classificado pelos fatores de correção de arco de contato e comprimento fornece o chamado HP efetivo.

→ Posteriormente, a divisão da potência de projeto pelo HP efetivo fornece, enfim, o número de correias a ser utilizado no projeto

$HP_{efet} = HP_{class}. F_{AC}.F_{LP}$

N° de correias $\geq \dfrac{P_P}{HP_{efet}}$

Obs.: Polia em V (motora) - Plana → $i \geq 0.3$  e $\dfrac{dP_2 - dp_1}{a_{corr}} \geq 0.7$

![](figuras/fig4-33.png)

![](figuras/fig4-34.png)

![](figuras/fig4-35.png)

![](figuras/fig4-36.png)

![](figuras/fig4-37.png)

![](figuras/fig4-38.png)

![](figuras/fig4-39.png)

![](figuras/fig4-40.png)

![](figuras/fig4-41.png)

![](figuras/fig4-42.png)

![](figuras/fig4-43.png)

![](figuras/fig4-44.png)

![](figuras/fig4-45.png)

![](figuras/fig4-46.png)

![](figuras/fig4-47.png)

![](figuras/fig4-48.png)


**Exemplo 2:**

Um britador de 29HP deverá ser acionado mediante uma transmissão por correias trapezoidais, em uma rotação de saída de 280RPM. O motor de acionamento apresenta 30HP a 1170RPM. Para a distância entre eixos aproximada de 40”, determinar o número e o tipo de correias trapezoidais para o acionamento referido. 

In [1]:
from sympy import *

Unidades:

In [2]:
pol,m,HP,W,RPM,s = symbols('pol m HP W RPM s')

Dados:

In [3]:
No = 30*HP # Potência nominal
n1 = 1170*RPM # Rotação do eixo motor
n2 = 280*RPM # Rotação do eixo movido
a = 40*pol # distância entrei eixos aproximada

Conversões:


In [4]:
conv = {pol:0.0254*m, HP:7368*W,RPM:1/(60*s)}

## Passo 01: Obter a potência de projeto

![image.png](figuras/fig4-49.png)

In [5]:
Ko = 1.5 # Fator de Serviço
Pp = Ko*No 
Pp # Potência de Projeto

45.0*HP

## Passo 02: Seleção do perfil da correia

![image.png](figuras/fig4-50.png)

Perfil C, polia menor -> 8" a 12"

## Passo 03: Definição do tamanho das polias:

Optando por polia motora (menor polia) de 10" (segundo a recomendação):

In [6]:
dp1 = 10*pol
i = n1/n2 # razão de transmissão
dp2 = dp1*i 
dp2

585*pol/14

In [7]:
dp2 = dp2.evalf(3)
dp2 

41.8*pol

## Passo 04 - Velocidade Tangencial

In [8]:
v = (pi*dp1*n1).subs(conv)
v.evalf(4)

15.56*m/s

## Passo 05 - Comprimento da Correia

In [9]:
L = 2*a + 1.57*(dp2 + dp1) + (dp2 - dp1)**2/(4*a)
L

167.625234375*pol

In [10]:
theta = pi - 2*asin((dp2-dp1)/(2*a))
theta.evalf(3)

2.32

## Passo 06:  
Obter o comprimento padronizado:

![image.png](figuras/fig4-51.png)

![image.png](figuras/fig4-52.png)

In [11]:
a_corr = 40*pol - (168.9*pol - L)/2 # distância entre centros corrigida
a_corr.evalf(4)

39.36*pol

## Passo 07
Fatores de correção de forma:

In [12]:
(theta*180/pi).evalf(5)

133.17

![image.png](figuras/fig4-53.png)

In [13]:
F_AC = 0.87

![image.png](figuras/fig4-54.png)

Interpolação linear:

$$\dfrac{166 - 158}{173 - 158} = \dfrac{ F_{LP} - 1,02}{1,04 - 1,02} $$

In [14]:
F_LP = (166-158)/(173-158)*(1.04-1.02) + 1.02
F_LP

1.0306666666666666

## Passo 08:
Obter o HP classificado

![image.png](figuras/fig4-55.png)

Interpolção linear:
$$ \dfrac{14.44 - 13.67} {1200 - 1100}= \dfrac{ HP_{bas} - 13.67}{1170 - 1100}$$

In [15]:
HP_bas = (14.44 - 13.67)/(1200 - 1100)*(1170 - 1100) + 13.67
HP_bas

14.209

In [16]:
i.evalf(2)

4.2

![image.png](figuras/fig4-56.png)

Interpolação linear:

$$\dfrac{2.10 - 1 .92}{1200 - 1100} = \dfrac{HP_{adic} - 1.92}{1170 - 1100} $$

In [17]:
HP_adic = (2.10 - 1.92)/(1200 - 1100)*(1170 - 1100)  + 1.92
HP_adic

2.0460000000000003

In [18]:
HP_class = (HP_bas + HP_adic)*HP
HP_class

16.255*HP

## Passo 09

Número de correias

In [19]:
HP_efet = HP_class*F_AC*F_LP
HP_efet

14.5755334*HP

In [20]:
No_de_correias = Pp/HP_efet
No_de_correias

3.08736557112894

In [21]:
No_de_correias = int(No_de_correias)+1
No_de_correias

4

## Especificação:

4 correias Perfil C - Continental 166

Distância entre centros = 39.36"

Diâmetro primitivo da polia motora = 10"

Diâmetro primitivo da polia movida = 41,8"